### DISCUSSION TITLE
how exactly do you install the newest version? +xbox question

### DISCUSSION THREAD

**Discussion by HELP0w0** [AvionBlock/VoiceCraft#107] [Q&A]
im not understanding it, there isnt enough detail, also if i where to install this in a world would my friends on xbox be able to use it too?

**Comment by Miniontoby** [comment]
# Installing newest version

You need to install NET8 drivers in order to host a VoiceCraft server (or you use Atrioxhosting, who provide servers for €0.44/month)

Also you should watch the Youtube Video for the instructions, but DON'T install NET7 drivers, install NET8 drivers!


# XBox players

If you want to use it on XBox as well, then Server Sided Positioning is your best bet.

You need to use Bedrock Dedicated Server software to host the world. and you need to install the addon on the server

Then for the XBox players, they need to also have a computer or android phone and then bind their player to VoiceCraft on their other device.
